src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/ResourceRoutes.scala

```diff
@@ -41,12 +41,12 @@ import monix.execution.Scheduler.Implicits.global
 import scala.concurrent.Future
 import scala.util.Try
 
-class ResourceRoutes(implicit repo: Repo[Task],
-                     adminClient: AdminClient[Task],
-                     iamClient: IamClient[Task],
-                     indexers: IndexerClients[Task],
-                     store: AttachmentStore[Task, AkkaIn, AkkaOut],
-                     config: AppConfig) {
+class ResourceRoutes(staticResolution: StaticResolution[Task])(implicit repo: Repo[Task],
```

Comment:
staticResolution can be a val in the  `StaticResolution` companion object. I don't see a reason why this is a class, since all the information necessary to create it it is available at compile time (we are not using anything coming form configuration)

```scala
object StaticResolution {
	val staticResolution = StaticResolution[Task](
	      Map(
	        tagCtxUri      -> "/contexts/tags-context.json",
	        resourceCtxUri -> "/contexts/resource-context.json"
	      ))
// other...
}
```

Reply:
or maybe `val staticResolution` should be placed on the `AppConfig` companion object. But in any case should be a `val` on some `object`